iothub/device/tests/ConnectionPoolTests.cs

```diff
@@ -14,225 +14,225 @@ namespace Microsoft.Azure.Devices.Client.Test
     [TestCategory("Unit")]
     public class ConnectionPoolTests
     {
-        [TestMethod]
```

Comment:
commenting out tests assuming pooling is work in progress.

Reply:
Correct

---
In reply to: [251639157](https://github.com/Azure/azure-iot-sdk-csharp/pull/770#discussion_r251639157) [](ancestors = 251639157)